lib/test.js

```diff
@@ -108,6 +109,12 @@ Test.prototype.test = function (name, opts, cb) {
 
 Test.prototype.comment = function (msg) {
     var that = this;
+
+    // Previous behavior involved `toString` calls on objects, i.e. emitting `[object Object]`.
```

Comment:
@ljharb Added a comment to explain the conditional

Reply:
This remains an issue - to restate, I think that we need something like `format.apply(null, Array.prototype.map.call(arguments, String))` to ensure that everything is always stringified.